site-main/variables.tf

```diff
@@ -3,6 +3,7 @@ variable region {
 }
 
 variable domain {}
+variable bucket_name {}
```

Comment:
Can you make the module backwards compatible by setting the default value of the bucket_name to the calculated value it was before?

` "site.${replace("${var.domain}",".","-")}"`